src/pages/react/navigation.md

```diff
@@ -97,6 +97,42 @@ The `DashboardPage` above shows a users list page and a details page. When navig
 
 An `IonRouterOutlet` should only contain `Route`s or `Redirect`s. Any other component should be rendered either as a result of a `Route` or outside of the `IonRouterOutlet`.
 
+## Fallback Route
+
+A common routing use case is to provide a "fallback" route to be rendered in the event the location navigated to does not match any of the routes defined. 
+
+We can define a route within an `IonRouterOutlet` to act as our "fallback" route -- provided no `path` is provided and it is the last `Route` defined.
```

Comment:
"provided no path is provided" sounds a bit odd since we use "provided" twice. Can we change to make clearer?